commands/core/site_install.drush.inc

```diff
@@ -287,6 +298,21 @@ function drush_core_site_install($profile = NULL) {
       $form_options[$key] = $value;
     }
   }
+
+  // If the profile is not explicitly set, default to the 'minimal' for an issue-free config import.
+  if (empty($profile) && drush_get_option('config')) {
+    $profile = 'minimal';
```

Comment:
It works for me either way, but the whole process is faster with 'minimal' - there is no time wasted installing things that the config import will override anyway.